Gemfile

```diff
@@ -27,6 +27,7 @@ gem 'turbolinks'
 
 # View
 gem 'simple_form'
+gem 'enumerize'
```

Comment:
@cawel Unfortunately `enumerize` is still included in this pull-request. In master it's located here:

https://github.com/montrealrb/Montreal.rb/blob/master/Gemfile#L23